.github/workflows/perf-device-models.yaml

```diff
@@ -36,9 +36,10 @@ jobs:
         run: |
           ./scripts/build_scripts/build_with_profiler_opt.sh
       - name: Run device performance regressions
-        timeout-minutes: 60
+        timeout-minutes: 90
         run: |
           source build/python_env/bin/activate
+          ./tests/scripts/run_profiler_regressions.sh PROFILER
           ./tests/scripts/run_tests.sh --tt-arch $ARCH_NAME --pipeline-type models_device_performance_${{ matrix.runner-info.machine-type }}
```

Comment:
@mo-tenstorrent Can you please put the script call inside the `models_device_performance_bare_metal` function call in `run_tests.sh` instead please?

Reply:
Yes  will do